pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -108,15 +104,15 @@ class Search extends Component<IAppState, ISearchState> {
         <DataList aria-label="Search results Headings" >
           <DataListItem aria-labelledby="span-name">
             <DataListItemRow id="data-rows-header" >
-              {this.props.userAuthenticated && !this.state.isEmptyResults &&
+              {/* {this.props.userAuthenticated && !this.state.isEmptyResults &&
```

Comment:
Are we sure, to remove checkboxes functionalities, if so, Can we delete this block, instead of commenting ?

Reply:
This is a Checkbox component that we can use when we implement builk editing on the search page.  That's why I commented it out instead of removing it.

Reply:
Cool !!